passes/techmap/extract_reduce.cc

```diff
@@ -144,9 +187,17 @@ struct ExtractReducePass : public Pass
 					{
 						if(!IsRightType(x, gt))
 							break;
+						if (!is_single_bit(x, sigmap))
+							break;
 
 						head_cell = x;
 
+						if (debug_verbose)
+						{
+							log("  Chain head cell is: %s\n", head_cell->name.c_str());
+							log("  Chain head cell Y width: %i\n", GetSize(head_cell->getPort("\\Y")));
```

Comment:
Can any of these Y width debug prints ever print a value other than 1? If not, there's no value in having these particular print statements (the other verbose logging is still useful).